.github/workflows/test.yml

```diff
@@ -31,6 +31,9 @@ jobs:
 
     name: Foundry project
     runs-on: ubuntu-latest
+    env:
+      FORK_URL: 'https://eth.llamarpc.com'
+      FORK_BLOCK_NUMBER: 20691292
```

Comment:
Perhaps a comment here that the fork URL should be an RPC that supports archive requests.